### PR TITLE
Update Mojang Studios

### LINKED ISSUE #3087: Update Mojang Studios
**Name:** Mojang Studios
**Official resources for icon and color:**
image

https://www.minecraft.net/article/meet-mojang-studios
https://twitter.com/Minecraft/status/1262050538897670145
https://www.youtube.com/watch?v=YosWmbHAr2g

### PR DESCRIPTION
**Pull request by fbernhart**
mojangstudios

Closes #3087 
**Alexa rank:** [13,959](https://www.alexa.com/siteinfo/mojang.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
The stacked version of the new logo can be found here: https://www.minecraft.net/content/dam/franchise/logos/Mojang-Studios-Logo-Redbox.png

@EinsEli vectorized this into a SVG file in #3087. I took his work, removed the color and resized and centered the icon.

### CODE REVIEW

**Comment by adamrusted** [comment]
There is an SVG of this available on the [Minecraft Gamepedia Wiki](https://minecraft.gamepedia.com/File:Mojang_Studios_logo.svg) - how usable that is as a source I'll leave up to @simple-icons/maintainers

**Comment by ericcornelissen** [comment]
> There is an SVG of this available on the [Minecraft Gamepedia Wiki](https://minecraft.gamepedia.com/File:Mojang_Studios_logo.svg) - how usable that is as a source I'll leave up to other @simple-icons/maintainers

I'd say it can be used as a starting point, but not as an official source (not all _-pedia_ sites have standards for this kind of stuff, though we could consider _gamedia_ for the future if it turns out they generally contain good SVGs). If it matches the official PNG then that's great!

File: icons/mojangstudios.svg
Comment (adamrusted) [inline icons/mojangstudios.svg:1]: <details>
<summary>There seem to be some differences between this (red) and the SVG source found (black).</summary>

</details>
<details>
<summary>Even with the PNG source (below), there are some small differences.</summary>

</details>
<details>
<summary>I've just pulled in the Gamepedia SVG, and it looks like the best possible match to the PNG file linked as source.</summary>

</details>

If there's no objection from any other @simple-icons/maintainers - I'd recommend that we use the **Gamepedia SVG** as our source - and can review at a later date, should the logo change and they can't keep up with changes.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/mojangstudios.svg
Comment (fbernhart) [reply icons/mojangstudios.svg]: @adamrusted Great work! I've now used the new SVG that you've provided.

Comment (adamrusted) [reply icons/mojangstudios.svg]: Fab - did you want to update the source to the SVG also?

Comment (fbernhart) [reply icons/mojangstudios.svg]: Quoting @ericcornelissen:

> I'd say it can be used as a starting point, but not as an official source

Comment (adamrusted) [reply icons/mojangstudios.svg]: Is the precision of 4 necessary on this?

Comment (fbernhart) [reply icons/mojangstudios.svg]: @adamrusted It's optimized with a precision of 3, isn't it? 🤔

Comment (adamrusted) [reply icons/mojangstudios.svg]: Yes. Don't mind me. Reading the preview from when I left the original comment 😅

**Review by adamrusted — APPROVED** [review]
LGTM! 🟩

**Pull request merged**
